src/layers/core/scatterplot-layer/scatterplot-layer-vertex.glsl

```diff
@@ -32,16 +32,22 @@ uniform float radius;
 uniform float radiusMinPixels;
 uniform float radiusMaxPixels;
 uniform float renderPickingBuffer;
+uniform float drawOutline;
+uniform float strokeWidth;
 
 varying vec4 vColor;
+varying vec2 uv;
+varying float strokeWidthRatio;
 
 void main(void) {
   // Multiply out radius and clamp to limits
   float radiusPixels = clamp(
     project_scale(radius * instanceRadius),
     radiusMinPixels, radiusMaxPixels
-  );
+  ) + drawOutline * strokeWidth / 2.0;
```

Comment:
> The outline stoke is centered at the radius

OK that is a reasonable choice. Probably still reasonable to mention it in the docs.

Also, is there a risk that a wide stroke overflows the containing square and gets truncated due to fragment shader not being called? Or are you scaling down the entire thing to make sure it fits?